cfg/windows.cfg

```diff
@@ -5098,6 +5098,70 @@ HFONT CreateFont(
       <not-uninit/>
     </arg>
   </function>
+  <!-- HLOCAL WINAPI LocalAlloc(
+  _In_ UINT   uFlags,
+  _In_ SIZE_T uBytes); -->
+  <function name="LocalAlloc">
```

Comment:
Did you leave out `<use-retval/>` by intention?

Reply:
Yes, `LocalAlloc` is already configured as `alloc` function and `<use-retval/>` would result in a redundant warning (see https://trac.cppcheck.net/ticket/8342)